azext_iot/central/services/user.py

```diff
@@ -43,16 +44,25 @@ def add_service_principal(
     """
     url = "https://{}.{}/{}/{}".format(app_id, central_dns_suffix, BASE_PATH, assignee)
 
+    if api_version == ApiVersion.v1.value:
+        user_type = (UserTypeV1.service_principal.value,)
```

Comment:
Are these values meant to be wrapped as tuples?

Reply:
fixed